demo/index.scss

```diff
@@ -11,3 +11,14 @@ $use-embedded-font: true;
 .container {
   margin: 20px;
 }
+
+
+//Example of how to style ws-tiles-chart using classes (.tile.groupName).
+.ws-tiles-chart .tile {
```

Comment:
The component contains its own SCSS file. This is just some CSS for the demo page to show that you can change the color of the tiles in two ways, 
1) Sending the config properties to the component
Like:   config: {
    groupName1:'#FF4D4D',
    groupName2: '#ffcc33'
  }
 2)using .tile.groupName in the SCSS of your application

If no SCSS is specified for the tile color and no config was sent to the component, by default, it will show the tiles as green for all the groups.